crypto/stack/stack.c

```diff
@@ -12,6 +12,34 @@
 #include "internal/numbers.h"
 #include <openssl/stack.h>
 #include <openssl/objects.h>
+#include <errno.h>
+
+/*
+ * The initial number of nodes in the array.
+ */
+static const int min_nodes = 4;
```

Comment:
I was meaning to ask why do you prefer static const over pre-processor macros? I'm not saying it's wrong/inappropriate, just curious over any particular reason. I can imagine it being handy in debugger, because you can examine the value [as opposite to remembering it or consulting likely another file]...

Reply:
Yes to debugging.  It also makes the compiler type check (which can be done in a macro but often isn't).  Compilers have optimised these away for ages now so there is no penalty either for performance or space.

I also think it's a bit neater this way.  I generally find expressions a little easier to read when they're mostly in lower case.

So no compelling reason for or against really.